dotCMS/src/integration-test/java/com/dotcms/publishing/job/TestCaseSiteSearchBuilder.java

```diff
@@ -0,0 +1,107 @@
+package com.dotcms.publishing.job;
+
+import com.dotcms.publishing.job.SiteSearchJobImplTest.TestCaseSiteSearch;
+
+public class TestCaseSiteSearchBuilder {
+
+    private boolean defaultPageToDefaultLanguage;
+    private boolean defaultContentToDefaultLanguage;
+    private boolean siteSearchDefaultLanguage;
+    private boolean siteSearchSecondLanguage;
+    private boolean siteSearchThirdLanguage;
+    private boolean createContentInDefaultLanguage;
+    private boolean createContentInSecondLanguage;
+    private boolean createContentInThirdLanguage;
+    private boolean createPageInDefaultLanguage;
+    private boolean createPageInSecondLanguage;
+    private boolean expectedResultsWhenSearchingContentInDefaultLanguage;
+    private boolean expectedResultsWhenSearchingContentInSecondLanguage;
+    private boolean expectedResultsWhenSearchingContentInThirdLanguage;
+
+    public TestCaseSiteSearchBuilder defaultPageToDefaultLanguage(
+            boolean defaultPageToDefaultLanguage) {
+        this.defaultPageToDefaultLanguage = defaultPageToDefaultLanguage;
+        return this;
+    }
+
+    public TestCaseSiteSearchBuilder defaultContentToDefaultLanguage(
+            boolean defaultContentToDefaultLanguage) {
+        this.defaultContentToDefaultLanguage = defaultContentToDefaultLanguage;
+        return this;
+    }
+
+    public TestCaseSiteSearchBuilder siteSearchDefaultLanguage(
+            boolean siteSearchDefaultLanguage) {
+        this.siteSearchDefaultLanguage = siteSearchDefaultLanguage;
+        return this;
+    }
+
+    public TestCaseSiteSearchBuilder siteSearchSecondLanguage(boolean siteSearchSecondLanguage) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'siteSearchSecondLanguage' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5303548)